src/utils/files.ts

```diff
@@ -1,4 +1,25 @@
-export const serializeDateForFileName = (date: Date) =>
-  `${date.getUTCFullYear()}-${
-    date.getUTCMonth() + 1
-  }-${date.getUTCDate()}-${date.getUTCHours()}-${date.getUTCMinutes()}`;
+export enum DateLevel {
+  Year = 0,
+  Month = 1,
+  Day = 2,
+  Hour = 3,
+  Minute = 4,
+  Second = 5,
+}
+
+export const serializeDateForFileName = (date: Date, level: DateLevel) =>
+  createDateParts(date)
+    .slice(0, level + 1)
+    .join("-");
+
+const createDateParts = (date: Date) =>
+  Object.values(
+    {
+      [DateLevel.Year]: date.getUTCFullYear().toString(),
```

Comment:
It's very unlikely, but if the date is smaller than 4 digits, then the number wouldn't have 4 digits. Is it ok?